internal/pally/counter.go

```diff
@@ -23,7 +23,9 @@ package pally
 import (
 	"fmt"
 
+	"github.com/golang/protobuf/proto"
```

Comment:
We're using `github.com/gogo/protobuf` for `encoding/x/protobuf`, might be nice to maintain symmetry here (the functionality is the same).